x-pack/plugins/event_log/server/es/cluster_client_adapter.ts

```diff
@@ -286,15 +286,13 @@ export class ClusterClientAdapter {
         hits: { hits, total },
       }: SearchResponse<unknown> = await this.callEs('search', {
```

Comment:
If you use the `ESSearchResponse` type you won't need to cast to `unknown`.

At the top add:
```
import { ESSearchResponse } from '../../../../typings/elasticsearch';
```
Then you can replace this with:

```suggestion
      }: ESSearchResponse<unknown, {}> = await this.callEs('search', {
```

Then below, replace the cast with:
```
        total: total.value,
```